Common/src/main/java/com/blamejared/crafttweaker/api/zencode/scriptrun/IScriptLoadingOptionsView.java

```diff
@@ -0,0 +1,30 @@
+package com.blamejared.crafttweaker.api.zencode.scriptrun;
+
+import com.blamejared.crafttweaker.api.ScriptLoadingOptions;
+
+import java.util.Collection;
+import java.util.stream.Stream;
+
+public interface IScriptLoadingOptionsView {
+    
+    boolean isFormat();
+    
+    boolean isExecute();
+    
+    String getLoaderName();
+    
+    Collection<String> getInheritedLoaders();
+    
+    ScriptLoadingOptions.ScriptLoadSource getSource();
+    
+    default Collection<String> getTargetedLoaders() {
+        
+        return Stream.concat(this.getInheritedLoaders().stream(), Stream.of(this.getLoaderName())).toList();
```

Comment:
Not that it makes a difference (since we return Collection), but should our loader be first or last in the returned list.
But since we return a list we have no order guarantees anyways.

Just in case `.distinct()` or `.toSet()`?